e2e-tests/src/test/performance.rs

```diff
@@ -152,7 +164,7 @@ async fn setup_test(
 
     let validator_keys = get_test_validators(config);
     let reserved_validators = account_ids_from_keys(&validator_keys.reserved);
-    let non_reserved_validators = account_ids_from_keys(&validator_keys.non_reserved);
+    let non_reserved_validators = account_ids_from_keys(&validator_keys.non_reserved).into_iter().take(2).collect::<Vec<_>>();
```

Comment:
nit: `account_ids_from_keys(&validator_keys.non_reserved)[..2].clone()` seems more idiomatic.

Reply:
Thanks for suggestion, but since it's a cosmetic change I'll leave it as is